src/util/__tests__/markdownToText.ts

```diff
@@ -164,18 +170,37 @@ __bold__
     const markdown = `
 1. First ordered item
 2. Second ordered item
-   * Unordered sub-list
+   * Unordered sub-list that collapses
 * Unordered item
 - Another unordered item
 + Yet another unordered item
+
+# The next list will collapse to this heading
+1. First ordered item
+2. Second ordered item
 `
     expect(markdownToText(markdown)).toBe(`
-- First ordered item
-- Second ordered item
-  - Unordered sub-list
-- Unordered item
-- Another unordered item
-- Yet another unordered item
+- ${' '}
```

Comment:
Thoughts are automatically trimmed, so using a space for the parent won't work. The `=scope` parent should be `::`.